pkg/network/network.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package network
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanchego/api/info"
+)
+
+type StatusChecker interface {
+	GetCurrentNetworkVersion() (string, int, error)
+}
+
+type networkStatusChecker struct{}
+
+func NewStatusChecker() StatusChecker {
+	return networkStatusChecker{}
+}
+
+func (networkStatusChecker) GetCurrentNetworkVersion() (string, int, error) {
```

Comment:
If the `string` returned is a semver version, maybe we could use that as the data type?

Reply:
I don't think there is a semver data type, so I don't think there's anything to do here.

Reply:
You're right, there isn't actually such a type